app/src/pages/playground/ModelConfigButton.tsx

```diff
@@ -111,20 +127,20 @@ function ModelConfigDialogContent(props: ModelConfigDialogContentProps) {
             });
           }}
         />
-        <ModelPicker
-          modelName={instance.model.modelName}
-          provider={instance.model.provider}
-          query={query}
-          onChange={(modelName) => {
-            updateModel({
-              instanceId: playgroundInstanceId,
-              model: {
-                provider: instance.model.provider,
-                modelName,
-              },
-            });
-          }}
-        />
+        {instance.model.provider === "AZURE_OPENAI" ? (
+          <TextField
+            label="Deployment Name"
```

Comment:
can have this as model, not sure what's easier for users, technically we need the deployment name

Reply:
I see